[ERROR] Trying to retrieve setting with no value given: 'max_feedrate_z_override'

I'm running into the following error? Can you help me? 

Thanks,
Joel

And even running the program with those extra parameters I get a new error:

```
[ERROR] Trying to retrieve setting with no value given: 'skin_alternate_rotation'
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-From ubuntu:16.04
+From ubuntu:18.04
 RUN apt-get -y update
 RUN apt-get -y install git wget autoconf automake libtool curl make g++ unzip cmake python3 python3-dev python3-sip-dev
 
@@ -14,10 +14,12 @@ RUN ./autogen.sh && ./configure && make && make install && ldconfig
 
 # install libArcus
 WORKDIR "/libArcus"
-RUN git checkout 4.0.0
+RUN git pull
+RUN git checkout 4.4
 RUN mkdir build && cd build && cmake .. && make && make install
 
 # install curaengine
 WORKDIR "/CuraEngine"
-RUN git checkout 4.0.0
+RUN git pull
+RUN git checkout 4.4
 RUN mkdir build && cd build && cmake .. && make

--- a/README.md
+++ b/README.md
@@ -14,10 +14,10 @@ Change dir to /CuraEngine/build
     $ cd /CuraEngine/build
     
 #### Get example configs and stl file
-    $ wget https://raw.githubusercontent.com/Ultimaker/Cura/master/resources/definitions/fdmprinter.def.json
-    $ wget https://raw.githubusercontent.com/Ultimaker/Cura/master/resources/definitions/fdmextruder.def.json
-    $ wget https://raw.githubusercontent.com/Ultimaker/Cura/master/resources/definitions/prusa_i3.def.json
-    $ wget https://raw.githubusercontent.com/fozbek/curaengine-dockerfile/master/herringbone-gear-large.stl
+    $ wget https://raw.githubusercontent.com/Ultimaker/Cura/4.4/resources/definitions/fdmprinter.def.json
+    $ wget https://raw.githubusercontent.com/Ultimaker/Cura/4.4/resources/definitions/fdmextruder.def.json
+    $ wget https://raw.githubusercontent.com/Ultimaker/Cura/4.4/resources/definitions/prusa_i3.def.json
+    $ wget https://raw.githubusercontent.com/KrisRoofe/curaengine-dockerfile/master/herringbone-gear-large.stl
 
 #### Slice It
     $ ./CuraEngine slice -p -j ./fdmprinter.def.json -j prusa_i3.def.json -l herringbone-gear-large.stl -o herringbone-gear-large.gcode

